discord/index.py

```diff
@@ -86,8 +86,8 @@ async def _check_called(ctx, state):
         await msg.delete(delay=5)
         return state
 help_dict = {
-            "**Top Scites**": "<@&846783290332413964>`top`", 
-            "**Now Scites**": "<@&846783290332413964>`top now`",
+            "**Start Scites**": "<@&846783290332413964>`start`", 
+            "**Now Scites**": "<@&846783290332413964>`start now`",
```

Comment:
hah, sorry for the nit-picking; i was thinking this could simply be `@now` instead of `start now`; then i think it's all perfect!

Reply:
no worries, tried my best to fix that. hopefully it works